src/services/affinidi/errors.ts

```diff
@@ -74,7 +74,7 @@ export function handleServiceError(
   const isServerError = isAxiosError && error.response && error.response.status >= 500
 
   if (isServerError) {
-    credentialsVault.clear()
+    void credentialsVault.clear()
```

Comment:
why is void needed here?

Reply:
Due to keytar methods are async, but it seems we don't need to wait here.